app/views/labs/show.html.erb

```diff
@@ -61,10 +61,10 @@
         </tr>
       </thead>
       <tbody>
-        <% @lab.lab_spaces.each do |ls| %>
+        <% @lab.lab_spaces.each do |lab_space| %>
```

Comment:
We could move the contents of this loop to a partial and use Rails' collection rendering, i.e.: 
```
<%= render @lab_spaces %>
```
But it's not a priority so we can do it later.